server/shared/termdb.usecase.js

```diff
@@ -179,6 +179,18 @@ export function isUsableTerm(term, _usecase, ds) {
 				return uses
 			}
 
+		case 'filter':
+			if (graphableTypes.has(term.type)) {
+				if (usecase.detail == 'GDC') {
```

Comment:
There is `ds` argument, maybe `GDC` should not be hardcoded. It should be `if (!ds.cohort?.excludedTermTypesByUseCase?.filter?.includes == 'gdc') uses.add('plot')`

Reply:
this can be hard. it intends ds to be server-side obj. here the relevant logic calls isUsable on client, so it's only able to pass in termdbconfig. and must change a dozen places isUsable is called on client